src/main/kotlin/net/maizegenetics/phgv2/cli/SetupEnvironment.kt

```diff
@@ -17,19 +19,38 @@ import java.util.stream.Collectors
  * creating it, do we assume it exists?
  */
 class SetupEnvironment : CliktCommand(help="create conda environment for PHG") {
-    // The output directory will store the log and error files from the ProcessBUilder conda create command
-    val outputDir by option("-o", "--outputDir", help = "Directory where ProcessBuilder, conda create env command log files will be written")
-        .required()
+
+    private val myLogger = Logger.getLogger("net.maizegenetics.phgv2.cli.SetupEnvironment")
+
+    val envFile by option("-e", "--env-file", help = "File containing the conda environment definition")
+        .default("")
 
     // This function uses ProcssBuilder to setup the phgv2 conda environment
     fun createEnvironment( envFile: String, outputDir:String) {
+
+        var selectedEnvFile = envFile
+        myLogger.info("begin run: selectedEnvFile = $selectedEnvFile")
+
+        // if no user defined environment file, use the default
+        if (selectedEnvFile == "") {
+
+            selectedEnvFile = "${outputDir}/environment.yml"
+            val inputUrl = SetupEnvironment::class.java.getResource("environment.yml")
```

Comment:
Is this doing the same thing as line 41?

Reply:
AH, yes!   Remnants of previous attempts to get this to work.  I'll delete line 38, it isn't needed.

Reply:
I wonder if there is a way to get the GitHub action release task to copy this file out and into the root directory so its in both the pig_v2.jar and in the working directory for the executable?  Might be worth investigating in the future.